urllib3/contrib/appengine.py

```diff
@@ -236,13 +236,21 @@ def _urlfetch_response_to_http_response(self, urlfetch_resp, **response_kw):
             encodings.remove('chunked')
             urlfetch_resp.headers['transfer-encoding'] = ','.join(encodings)
 
+        original_response = HTTPResponse(
+          # In order for decoding to work, we must present the content as
+          # a file-like object.
+          body=BytesIO(urlfetch_resp.content),
+          msg=urlfetch_resp.header_msg,
+          headers=urlfetch_resp.headers,
+          status=urlfetch_resp.status_code,
+          **response_kw)
```

Comment:
The indentation here is off: can you indent another two spaces?

Reply:
Also can you match the house style of having the closing parenthesis on its own line?